### PR TITLE
docs updated for 2 13 release

### PR DESCRIPTION
**Pull request by osandoval011**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [ ] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [ ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [ ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [ ] My PR has been functionally tested.
- [ ] All of the [unit-tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) still pass.
- [ ] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: _docs/_latest/setup/upgrade.md [hunk 1]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+1. Remove the compute engine section from the deployment template.
+1. Run the deployment manager uppdate command on the updated deployment template.  
+`gcloud deployment-manager deployments update DEPLOYMENT_NAME --config forseti_server_v2_x_x.yaml`  
+This will remove your VM instance.
+1. Paste the compute engine section back to the deployment template, and run the update command again.  
+`gcloud deployment-manager deployments update DEPLOYMENT_NAME --config forseti_server_v2_x_x.yaml`  
+This will recreate the VM with updated fields.
+
+{% endcapture %}
+{% include site/zippy/item.html title="Error while running deployment manager" content=deployment_manager_error uid=50 %}
```
Comment (osandoval011) [inline _docs/_latest/setup/upgrade.md]: what does uid=50 signify here? @red2k18

File: _docs/_latest/setup/upgrade.md [hunk 2]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.X installations" content=2x_upgrade uid=1 %}
```
Comment (osandoval011) [inline _docs/_latest/setup/upgrade.md]: @blueandgold, @red2k18  What should I put for whats next?

File: _docs/_latest/setup/upgrade.md [hunk 3]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+copy the deployment name for Forseti server.
+1. Run command `gcloud deployment-manager deployments update DEPLOYMENT_NAME --config deploy-forseti-server-xxxxx-2-13-0.yaml`
+If you see errors while running the deployment manager update command, please refer to below section
+`Error while running deployment manager` for details on how to workaround the error.
+1. Reset the Forseti server VM instance for changes in startup script to take effect. 
+You can reset the VM by running command `gcloud compute instances reset MY_FORSETI_SERVER_INSTANCE --zone MY_FORSETI_SERVER_ZONE` 
+Example command: `gcloud compute instances reset forseti-server-vm-70ce82f --zone us-central1-c`
+1. Repeat step `3-9` for Forseti client.
+1. Configuration file `forseti_conf_server.yaml` updates: 
+   **Inventory**
```
Comment (red2k18) [inline _docs/_latest/setup/upgrade.md]: This is not applicable to current release. Remove the inventory part pls. There are no changes in the config file for this release.

File: _docs/_latest/setup/upgrade.md [hunk 4]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+            #
+            #composite_root_resources:
+            #    - "folders/12345"
+            #    - "folders/45678"
+            #    - "projects/98765"
+            #    - "organizations/56789"
+
+       ```
+
+1. Rule files updates:
```
Comment (red2k18) [inline _docs/_latest/setup/upgrade.md]: KMS rules file is not changing for this release. Pls remove it.

File: _docs/_latest/setup/upgrade.md [hunk 5]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+  
+### Steps to upgrade using Terraform
+
+1. Update the version inside main.tf file to 1.3.0.
+1. Run command terraform init to get the plugins.
+1. Run command terraform plan to see the infrastructure plan.
+1. Run command terraform apply to apply the infrastructure build.
+
+{% endcapture %}
+{% include site/zippy/item.html title="Upgrading 2.12.0 to 2.13.0" content=upgrading_2_12_0_to_2_13_0 uid=13 %}
```
Comment (red2k18) [inline _docs/_latest/setup/upgrade.md]: change uid to 14.

File: _docs/_latest/setup/upgrade.md [hunk 6]
```diff
@@ -1200,6 +1200,127 @@ This will recreate the VM with updated fields.
...
+
```
Comment (red2k18) [inline _docs/_latest/setup/upgrade.md]: This remains same for all the releases. Pls remove this.

File: _docs/_latest/setup/upgrade.md [hunk 7]
```diff
@@ -1200,6 +1200,89 @@ This will recreate the VM with updated fields.
...
+1. Update the version inside main.tf file to 1.3.0.
+1. Run command terraform init to get the plugins.
+1. Run command terraform plan to see the infrastructure plan.
+1. Run command terraform apply to apply the infrastructure build.
+
+{% endcapture %}
+{% include site/zippy/item.html title="Upgrading 2.12.0 to 2.13.0" content=upgrading_2_12_0_to_2_13_0 uid=14 %}
+
+{% capture deployment_manager_error %}
+If you get the following error while running the deployment manager:
```
Comment (red2k18) [inline _docs/_latest/setup/upgrade.md]: This is applicable to all releases. Pls remove it.

File: _docs/_latest/setup/upgrade.md (hunk 2 above)
Comment (red2k18) [reply _docs/_latest/setup/upgrade.md]: You don't have to edit that section. It remains the same.

**Pull request merged**